homework/index.js

```diff
@@ -30,13 +30,127 @@
     return elem;
   }
 
+  function buildRepositoryInfo(repositoryObject, containerDiv) {
+    const repositoryTable = createAndAppend('table', containerDiv, {
+      id: 'repository-table',
+    });
+    const repositoryTableBody = createAndAppend('tbody', repositoryTable, {});
+    const repositoryTableRow1 = createAndAppend('tr', repositoryTableBody, {});
+    createAndAppend('th', repositoryTableRow1, { text: 'Repository:' });
+    const tableName = createAndAppend('td', repositoryTableRow1, {});
+    createAndAppend('a', tableName, {
+      text: repositoryObject.name,
+      href: repositoryObject.html_url,
+      target: '_blank',
+    });
+    const repositoryTableRow2 = createAndAppend('tr', repositoryTableBody, {});
+    createAndAppend('th', repositoryTableRow2, { text: 'Description:' });
+    createAndAppend('td', repositoryTableRow2, { text: repositoryObject.description });
+    const repositoryTableRow3 = createAndAppend('tr', repositoryTableBody, {});
+    createAndAppend('th', repositoryTableRow3, { text: 'Forks:' });
+    createAndAppend('td', repositoryTableRow3, { text: repositoryObject.forks_count });
+    const repositoryTableRow4 = createAndAppend('tr', repositoryTableBody, {});
+    createAndAppend('th', repositoryTableRow4, { text: 'Last updated:' });
+    createAndAppend('td', repositoryTableRow4, { text: repositoryObject.updated_at });
+  }
+
+  function buildRepositoryContributors(contributorsArray, containerDiv) {
```

Comment:
Because you use bracket notation inside your function it is already clear the first argument is an array, so I would leave out the suffix `Array`.